docs/mine-hnt/validators/troubleshooting.mdx

```diff
@@ -30,26 +30,28 @@ Syncing to the blockchain could be impacted for the following reasons:
 * you just launched an AMI instance (if using AWS)
 
 ## Grab the latest blessed snapshot from Helium
-* On a running Validator run `miner snapshot list` or you can use the API to get the latest blessed snapshot going to this web address `https://api.helium.io/v1/snapshots`.
-* Find the highest "Height" block and download the snapshot from Helium via `curl -O https://snapshots.helium.wtf/mainnet/snap-<height>`. If this snapshot is not available, try the next highest as the consensus group may have not agreed upon the snapshot.
+
+NOTE: If you build the Validator from source, the paths below may be different for you. For the following Docker example, `/var/data` is inside the docker container which is mapped to your `validator_data` directory on the host system and `validator_data` is in your $HOME directory.  
+
+* Use the API to determine the latest [automatically blessed snapshot](https://api.helium.io/v1/snapshots) from `https://api.helium.io/v1/snapshots` or the latest [manually blessed snapshot](https://snapshots.helium.wtf/mainnet/latest-snap.json) from `https://snapshots.helium.wtf/mainnet/latest-snap.json`
```

Comment:
I suggest some additional clarification on the sources of the snapshots, maybe something like this:

Helium hosts snapshots for download for the convenience of syncing validators. You may download snapshot by following these steps
* First determine the height of the snapshot you would like to download:
  * The list of latest snapshot which were agreed upon by the consensus group is available on the [API](https://api.helium.io/v1/snapshots) from `https://api.helium.io/v1/snapshots`
  * Additionally, the latest snapshot used by hotspots to sync is also available from `https://snapshots.helium.wtf/mainnet/latest-snap.json`(https://snapshots.helium.wtf/mainnet/latest-snap.json). Note, these latest snapshot may include snapshots generated by Helium or the Helium ETL and are not agreed upon by the Consensus Group and may not have been tested to work on validators.
* `curl....`